api/src/main/java/marquez/common/models/Field.java

```diff
@@ -36,7 +36,7 @@
   @Getter
   private final FieldName name;
 
-  @Getter private final FieldType type;
+  @Nullable @Getter private final String type;
```

Comment:
should we apply the same for SchemaField to keep them in sync?

Reply:
We'll eventually want to use the classes published by OpenLineage, but yeah to keep the models in sync I can also update the class `SchemaField`.